app/assets/javascripts/hidden_until_select.js.coffee

```diff
@@ -0,0 +1,16 @@
+class HiddenUntilSelect
+  constructor: ($root, $hidden_el) ->
+    @$hidden_el = $hidden_el
+    @$controller = $root.find("##{ $hidden_el.attr('data-hide-until-select') }")
+    @$controller.keyup => @checkHide()
+    @$controller.change => @checkHide()
+    @checkHide()
+
+  checkHide: ->
+    @$hidden_el.toggle(@$controller.val())
+
+$ ->
+  # @todo - better scope
+  $scope = $(document)
+  $scope.find("[data-hide-until-select]").each (idx, el) ->
+    new HiddenUntilSelect($scope, $(el))
```

Comment:
Not necessarily a blocker, but this whole class seems like very similar functionality to the existing [Filter](https://github.com/18F/C2/blob/master/app/assets/javascripts/filter.js.coffee)...we should look to combine them.